memor/keywords.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+"""Tokens estimator keywords."""
+
+COMMON_PREFIXES = {"un", "re", "in", "dis", "pre", "mis", "non", "over", "under", "sub", "trans"}
+
+COMMON_SUFFIXES = {"ing", "ed", "ly", "es", "s", "ment", "able", "ness", "tion", "ive", "ous"}
+
+PYTHON_KEYWORDS = {"if", "else", "elif", "while", "for", "def", "return", "import", "from", "class",
+                   "try", "except", "finally", "with", "as", "break", "continue", "pass", "lambda",
+                   "True", "False", "None", "and", "or", "not", "in", "is", "global", "nonlocal"}
+
+JAVASCRIPT_KEYWORDS = {"if", "else", "switch", "case", "default", "for", "while", "do", "break",
+                       "continue", "function", "return", "var", "let", "const", "class", "extends",
+                       "super", "import", "export", "try", "catch", "finally", "throw", "new",
+                       "delete", "typeof", "instanceof", "in", "void", "yield", "this", "async",
+                       "await", "static", "get", "set", "true", "false", "null", "undefined"}
+
+JAVA_KEYWORDS = {"if", "else", "switch", "case", "default", "for", "while", "do", "break",
+                 "continue", "return", "void", "int", "float", "double", "char", "long", "short",
+                 "boolean", "byte", "class", "interface", "extends", "implements", "new", "import",
+                 "package", "public", "private", "protected", "static", "final", "abstract",
+                 "try", "catch", "finally", "throw", "throws", "synchronized", "volatile", "transient",
+                 "native", "strictfp", "assert", "instanceof", "super", "this", "true", "false", "null"}
+
+
+C_KEYWORDS = {"if", "else", "switch", "case", "default", "for", "while", "do", "break", "continue",
+              "return", "void", "char", "int", "float", "double", "short", "long", "signed",
+              "unsigned", "struct", "union", "typedef", "enum", "const", "volatile", "extern",
+              "register", "static", "auto", "sizeof", "goto"}
+
+
+CPP_KEYWORDS = {"if", "else", "switch", "case", "default", "for", "while", "do", "break",
```

Comment:
We can extend the `C_KEYWORDS` for `CPP_KEYWORDS`.

Reply:
fixed in https://github.com/openscilab/memor/pull/94/commits/c2b7cffa6f32b02edfd15f54382011c68b63ef0b